nemo/collections/tts/models/aligner.py

```diff
@@ -99,11 +104,10 @@ def _setup_tokenizer(self, cfg):
         text_tokenizer_kwargs = {}
         if "g2p" in cfg.text_tokenizer:
             # for backward compatibility
-            if self._is_model_being_restored() and cfg.text_tokenizer.g2p.get('_target_', None):
```

Comment:
as an FYI, I remvoed `cfg.text_tokenizer.g2p.get('_target_', None)` because "AttributeError: 'NoneType' object has no attribute 'replace'"

Reply:
Sure, as long as we're certain a config won't have this missing.

Reply:
hmm...`_target_` should be a required argument for `hydra.instantiate()`.